src/components/molecules/event/eventBody/EventBody.tsx

```diff
@@ -13,40 +13,49 @@ import {
   descriptionValidator,
   maxParticipantsValidator,
   titleValidator,
+  priceValidator,
+  timeValidator,
 } from 'utils/validators';
 import useForm from 'hooks/useForm';
 import { useHistory } from 'react-router-dom';
 import { Event } from 'models/apiModels';
 import { RoleOptions } from 'contexts/authProvider';
-
 // TODO extend the admin features
 export const EditEvent: React.FC<{ event: Event; setEdit: () => void }> = ({
   event,
   setEdit,
 }) => {
   const [error, setError] = useState<string | undefined>(undefined);
   const history = useHistory();
+  const timeDate = new Date(event.date);
+  const [toggleActive, setToggleActive] = useState<boolean>(
+    event.active ?? false
+  );
+  const [toggleFood, setToggleFood] = useState<boolean>(event.food ?? false);
+  const [toggleTransportation, setToggleTransportation] = useState<boolean>(
+    event.transportation ?? false
+  );
   const initalValue = {
     title: event.title,
     description: event.description,
-    date: event.date,
+    date: event.date.split('T')[0], //sets [YYYY-MM-DD]
+    time: timeDate.getHours() + ':' + timeDate.getMinutes(), //sets [HH:MM] and not [HH:MM:SS]
     address: event.address,
+    price: event.price.toString(),
```

Comment:
Is there any reason for converting the price to string?

Reply:
`export const priceValidator = (price: string) => {
  return price.length ? undefined : ['Arrangement pris må fylles ut'];
};`
Pricevalidator validates price as `String`